backend/api/v2beta1/go_client/experiment.pb.go

```diff
@@ -111,6 +111,8 @@ type Experiment struct {
 	Namespace string `protobuf:"bytes,5,opt,name=namespace,proto3" json:"namespace,omitempty"`
 	// Output. Specifies whether this experiment is in archived or available state.
```

Comment:
should this file be merged in? It seems to be a go source file generated by the protobuf.

Reply:
Looks like it needs to be merged, ignore.